frontend/common/src/components/Breadcrumbs/v2/Breadcrumbs.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { useIntl } from "react-intl";
+
+import Flourish from "../../Flourish";
+
+import Crumb from "./Crumb";
+
+export interface BreadcrumbsProps {
+  crumbs: {
+    label: React.ReactNode;
+    url: string;
+  }[];
+}
+
+const Breadcrumbs = ({ crumbs }: BreadcrumbsProps) => {
```

Comment:
Could we get a story for this new component?

Reply:
Added in [add stories for breadcrumbs + hero](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4293/commits/310aca756461a738206a8b53f7725df6b56eb1be)